mandatory/4-tax.js

```diff
@@ -17,7 +24,14 @@ function calculateSalesTax() {}
   Remember that the prices must include the sales tax (hint: you already wrote a function for this!)
 */
 
-function addTaxAndFormatCurrency() {}
+function addTaxAndFormatCurrency(price) {
+  const totalPrice = price * 1.2;
```

Comment:
This looks very good! But one thing you could try here is calling your **calculateSalesTax** function which you wrote above. It's doing the calculation for you, so you don't need to repeat that logic here.